src/Umbraco.Cms.Api.Management/Controllers/RelationType/AllRelationTypeTreeController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Common.ViewModels.Pagination;
+using Umbraco.Cms.Api.Management.ViewModels.RelationType;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Api.Management.Controllers.RelationType;
+
+[ApiVersion("1.0")]
+public class RootRelationTypeTreeController : RelationTypeControllerBase
```

Comment:
Should probably be called `AllRelationTypeController` to align with languages? As a bare minimum the file name should match the class name 😄